pages/index.js

```diff
@@ -1,10 +1,20 @@
 import PlantCard from "/components/PlantCard";
 import Link from "next/link";
+import styled from "styled-components";
 
 export default function HomePage({ handleToggleOwned, plants }) {
+
   return (
     <main>
       <h1>Plant List</h1>
+      { plants.length === 0 ? (
+        <>
+          <StyledInfoText>No plants there yet. Add new ones!</StyledInfoText>
+          <br />
+          <StyledButton type="button">Add Plant</StyledButton>
+          <br />
+        </>
+      ) : (
```

Comment:
Great solution! This way the user is clearly informed about what's going on together with a call to action so he can proceed and won't get stuck.

Short hint: I saw a few `<br />` in your code. Consider working with `margin` and/or `padding`. This way you have full control about the spacing.